cli/src/commands/git.rs

```diff
@@ -1065,3 +1067,51 @@ pub fn cmd_git(
         }
     }
 }
+
+pub fn git_init(
```

Comment:
nit: insert before `cmd_git()` (or around `cmd_git_fetch()`/`cmd_git_clone()`?)
Apparently, functions are sorted that way in this module.

Reply:
I didn't find a completely satisfactory sorting, so I've opted to put it just before cmd_git_import(). I'll be adding cmd_git_init() there, so it will end up in front of it's caller.

I'm calling good enough ;-)

Reply:
On second thought, I moved it above cmd_git_fetch(), so I can insert cmd_git_init() there eventually.